epicure-frontend/src/pages/home/components/Footer/Footer.scss

```diff
@@ -1,16 +1,61 @@
 @import "../../../../data/types/typography.scss";
 @import "../../../../data/variables.scss";
+@import "../../../../data/UI/colors.scss";
 
-.homePage {
+
+footer {
     display: flex;
-    //flex-direction: column;
-    padding-left: 20px;
-    padding-right: 20px;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    height: 202px;
+    top: 3763px;
+    gap: 0px;
+    opacity: 0px;
+
 
     @media (min-width: $desktop-viewport-width) {
-        padding-left: 127px;
-        padding-right: 115px;
+        height: 84px;
+        padding-bottom: 21px;
+        gap: 0px;
+        opacity: 0px;
+        align-items: center;
+        
+        
     }
 
-              
+    .footer-links {
+        display: flex;
+        flex-direction: column;
+        width: Hug (126px)px;
+        height: Hug (146px)px;
```

Comment:
same in here

Reply:
done, didn't effected the sale so I deleted them